timApp/velp/velp.py

```diff
@@ -91,7 +91,7 @@
 
 
 @velps.get("/<int:doc_id>/get_default_velp_group")
-def get_default_velp_group(doc_id: int):
+def get_default_velp_group(doc_id: int) -> Response:
```

Comment:
Hyvä, että tyypittää samalla nämä "vanhat" tiedostot. Kannattaa silloin samalla kokeilla kytkeä luokalle MyPy-tarkistus päälle ottamalla pois ignore `pyproject.toml`:sta:

https://github.com/TIM-JYU/TIM/blob/9654178c98e7e36e35a4f09e14bcef87faed9cfd/pyproject.toml#L300

ja ajamalla MyPy lokaalisti. Se sitten listaa sinulle vielä puuttuvat tyypitykset (tai sitten tyyppivirheet), jotka voi korjata. Näitä voi sitten pikkuhiljaa korjata joko muiden PRien yhteydessä tai sitten tekemällä ihan oma PR. Näin saadaan joskus ehkä kaikki tyypitettyä oikein :)

Ehdotan ainakin nyt kokeilemaan kytkeä ignore pois tälle tiedostolle ja ajaa MyPy-tarkistus lokaalisti. Jos tästä tiedostosta ei puutu kovin paljoa, niin tyypityksen voi hoitaa loppuun. Jos on paljon MyPy-virheitä, niin ei tähän PRään ole pakko. Sitten tyypitys voi korjata omana PRnä.

Reply:
Joo, olin unohtanut, että MyPyn ignoret löytyvät tuolta pyroject.tomlista. Kyllähän sieltä ihan hyvä läjä valituksia tulee, voi olla järkevämpi korjailla omana PR:änä.